src/transaction/mvcc_table.cpp

```diff
@@ -495,6 +495,7 @@ mvcctable::complete_mvcc (int tran_index, MVCCID mvccid, bool committed)
        *
        * It will be set to NULL after LOG_COMMIT
        */
+      assert (tran_index < m_transaction_lowest_visible_mvccids_size);
```

Comment:
Nit: I think is wiser to isolate the changes from this file in separate PR.

Reply:
Reverted, will be done separately.